src/components/common/YourCompany/CompanyName/CorrectNameRequest.vue

```diff
@@ -100,7 +98,7 @@ export default class CorrectNameRequest extends Mixins(CommonMixin, SharedMixin,
   protected applicantPhone = ''
   protected applicantEmail = ''
 
-  // *** FUTURE: use this to turn on/off validations
```

Comment:
I use `***` as an immediate must-fix thing (ie, before merging a PR). This is not one of those things.